src/components/navbar/navbar.scss

```diff
@@ -30,4 +30,9 @@
   &:not(:last-child) {
     margin-right: 2em;
   }
-}
\ No newline at end of file
+}
+@media screen and (max-width: 600px) {
```

Comment:
Please use the `respond-to` https://github.com/Coding-Coach/coding-coach/blob/development/src/theme/base/_mediaqueries.scss#L36